extensions/machine-learning-services/src/modelManagement/azureModelRegistryService.ts

```diff
@@ -21,6 +21,7 @@ import { HttpClient } from '../common/httpClient';
 import * as UUID from 'vscode-languageclient/lib/utils/uuid';
 import * as path from 'path';
 import * as os from 'os';
+import * as utils from '../common/utils';
```

Comment:
Are relative file imports considered best practice for ADS?

Reply:
I think for extensions is fine. for core we have the roots (sql, vscode) so it's better to put the root so it's easier to figure out which one is referenced.  

---
In reply to: [386568418](https://github.com/microsoft/azuredatastudio/pull/9377#discussion_r386568418) [](ancestors = 386568418)